CONTRIBUTING.md

```diff
@@ -205,7 +205,7 @@ that the review process goes smoothly.
   upcoming release. The format of changelog entries **must** match the template
   in [changelog/README.md](changelog/README.md). **Note:** If a changelog entry
   is required, a project maintainer will inform you on your pull request.
-* Run [clang-format-9](https://clang.llvm.org/docs/ClangFormat.html) on any new
+* Run [clang-format](https://clang.llvm.org/docs/ClangFormat.html) on any new
```

Comment:
should we say clang-format-16 here?

Reply:
I've updated this in the latest commit push.